### PR TITLE
moving AppenderContext to watchdog api

### PR DESCRIPTION
**Pull request by CuriousVini**
Issue: https://issues.cask.co/browse/CDAP-8480
Build: http://builds.cask.co/browse/CDAP-RUT628-1

### CODE REVIEW

File: cdap-watchdog-api/pom.xml [hunk 1]
```diff
@@ -53,6 +53,10 @@
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
     </dependency>
+    <dependency>
```
Comment (chtyim) [inline cdap-watchdog-api/pom.xml]: Move it before `junit` as `junit` is a test dependency while this one is not.

**Comment by chtyim** [comment]
Just one comment in the pom, rest LGTM.

File: cdap-watchdog-api/pom.xml (hunk 1 above)
Comment (CuriousVini) [reply cdap-watchdog-api/pom.xml]: done

**Comment by chtyim** [comment]
LGTM. Squash and merge.

**Comment by CuriousVini** [comment]
Build is green. squashing and merging

**Pull request merged**